chain/vm/Cargo.toml

```diff
@@ -39,4 +39,6 @@ version = "=0.11.0"
 path = "../core"
 
 [dependencies.multiversx-chain-vm-executor]
-version = "0.2.0"
+# version = "0.2.0"
```

Comment:
version 0.3.0 is what you need
point this PR to branch `feat/vm-1.7.next1` and merge from there, it's already integrated